presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeWindowQueries.java

```diff
@@ -50,23 +55,58 @@ protected void createTables()
             "PARTITION BY custkey, orderkey, shippriority",
             "PARTITION BY orderkey, custkey");
 
-    protected List<String> getRankingQueries(String rankingFunction, boolean orderBy)
+    private static final List<String> FRAME_CLAUSES = Arrays.asList(
+            // Frame clauses in RANGE mode.
+            "RANGE UNBOUNDED PRECEDING",
+            "RANGE CURRENT ROW",
+            "RANGE BETWEEN UNBOUNDED PRECEDING and UNBOUNDED FOLLOWING",
+
+            // Frame clauses in ROWS mode.
+            "ROWS UNBOUNDED PRECEDING",
+            "ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING",
+            "ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING",
+            "ROWS BETWEEN 5 PRECEDING AND CURRENT ROW",
+            "ROWS BETWEEN 5 PRECEDING AND UNBOUNDED FOLLOWING",
+            "ROWS BETWEEN CURRENT ROW AND 5 FOLLOWING",
+            "ROWS BETWEEN UNBOUNDED PRECEDING AND 5 FOLLOWING",
+            "ROWS BETWEEN orderkey PRECEDING AND CURRENT ROW",
+            "ROWS BETWEEN orderkey PRECEDING AND UNBOUNDED FOLLOWING",
```

Comment:
ROWS BETWEEN orderkey PRECEDING AND UNBOUNDED FOLLOWING
and ROWS BETWEEN UNBOUNDED PRECEDING AND orderkey FOLLOWING are not needed.